source/about-mongosync.txt

```diff
@@ -157,6 +157,28 @@ If the reverse sync is successful, ``mongosync`` enters the ``RUNNING`` state.
 The synchronization continues in the reverse direction from the original 
 sync job. 
 
+.. _mongosync-considerations:
+
+Considerations for Continuous Sync
+----------------------------------
+
+Consider the following when using ``mongosync`` for continuous synchronization:
+
+- For the destination cluster to have an up to date, consistent view of the 
+  source cluster’s data, ``mongosync`` must :ref:`commit <c2c-api-commit>` . 
+  If the source cluster shuts down before ``mongosync`` can commit, such as in 
+  a disaster scenario, the destination cluster might not have a consistent 
+  snapshot of the source data, and transactions might not appear atomically on 
+  the destination cluster.
+  
+  - For any continuous synchronization use cases with ``mongosync``, ensure that 
+    ``mongosync`` commits before cutting over from the source to the 
+    destination.
```

Comment:
Would be good to also add, "note that there is no way to resume the continuous sync between the two clusters once commit occurs, since mongosync can only sync into empty destination clusters. Use a fresh destination cluster to start a new continuous sync if desired." CC @rohshar

Reply:
Added a note!